main.tf

```
@@ -113,11 +113,10 @@ module "database" {
 # TFE and Replicated settings to pass to the tfe_init module
 # -----------------------------------------------------------------------------
 module "settings" {
-  source = "git::https://github.com/hashicorp/terraform-random-tfe-utility//modules/settings?ref=main"
+  source = "git::https://github.com/hashicorp/terraform-random-tfe-utility//modules/settings?ref=tf-440"
```

This one is finished testing now.
```suggestion
  source = "git::https://github.com/hashicorp/terraform-random-tfe-utility//modules/settings?ref=main"
```